sdk/attestation/azure-security-attestation/tests/test_attestation.py

```diff
@@ -247,29 +254,31 @@ def _test_attest_sgx_enclave(self, base_uri):
         quote = oe_report[16:]
         runtime_data = base64url_decode(_runtime_data)
         response, _ = attest_client.attest_sgx_enclave(quote, runtime_data=runtime_data)
-        assert response.enclave_held_data == runtime_data
-        assert response.sgx_collateral is not None
-
-        # Now do the validation again, this time specifying runtime data as JSON.
-        response, _ = attest_client.attest_sgx_enclave(quote, runtime_json=runtime_data)
-        # Because the runtime data is JSON, enclave_held_data will be empty.
-        assert response.enclave_held_data == None
-        assert response.runtime_claims.get("jwk") is not None
-        assert response.runtime_claims["jwk"]["crv"] == "P-256"
-        assert response.sgx_collateral is not None
-
-        # Call into the attest API asking it to *not* validate the token.
-        response, _ = attest_client.attest_sgx_enclave(
-            quote, runtime_data=runtime_data, validate_token=False
-        )
+        # assert response.enclave_held_data == runtime_data
```

Comment:
Why remove these tests? 

If the tests are not valid, then just remove the code, don't comment it out.

Reply:
This was just for debugging through to fix the errors, the PR is still not fully ready for review